tools/app/app/api/cards/[key]/route.tsx

```diff
@@ -240,11 +240,25 @@ async function getCardDetails(
       })
       .toString();
 
+    // always parse for now
+    await commands.calculateCmd.generate();
+
+    const card = await commands.calculateCmd.runQuery('card', {
+      cardKey: key,
+    });
+
+    if (card.length !== 1) {
+      return new NextResponse(`Query failed. Check card-query syntax`, {
+        status: 500,
+      });
+    }
+
     if (cardDetailsResponse) {
       return NextResponse.json({
-        ...cardDetailsResponse,
+        ...card[0],
```

Comment:
Update also the response changes to the swagger documentation at the start of this function